platform/server/http/Proxy/Proxy.ts

```diff
@@ -0,0 +1,84 @@
+import { Server } from 'net';
+import { IncomingMessage, OutgoingMessage } from 'http';
+import { EventEmitter } from 'events';
+
+import { Logger } from '../../../logging';
+
+export class Proxy extends EventEmitter {
+  private server: Server;
+  private readonly eventHandlers: any;
+  private port: number;
```

Comment:
Isn't specified in the constructor, so should be:

```
private port?: number;
```

(Also: I'm hoping they implement https://github.com/Microsoft/TypeScript/issues/8476 soon)

Reply:
✔️